src/app/components/ThemeProvider/focusIndicator.ts

```diff
@@ -7,7 +7,8 @@ const focusIndicator = ({ palette }: Theme) => css`
   // Adds focus indicator styling to all a and button elements by default.
   a:focus-visible,
   button:focus-visible,
-  button[type='submit']:focus-visible {
+  button[type='submit']:focus-visible,
+  h3:focus-visible {
```

Comment:
Do we want this for all h3's across the site?

Reply:
This should be fine as h3's aren't interactive elements, so they would only be able to be programmatically focused on https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex#:~:text=Some%20focusable%20HTML,focus%20navigation%20order).

Reply:
Ok, was just checking that you meant to do this for all h3s